<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramEqualizationFilterInterface</class>
 <widget class="QWidget" name="HistogramEqualizationFilterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>213</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="thresholdGroupBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>30</y>
     <width>202</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Threshold</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>112</y>
      <width>193</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="lowerHLayout_bottom"/>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>41</y>
      <width>193</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="upperHLayout_bottom"/>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>193</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="upperHLayout_top">
     <item>
      <widget class="QLabel" name="upperLabel">
       <property name="text">
        <string>Upper:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="upperHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>91</y>
      <width>193</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="lowerHLayout_top">
     <item>
      <widget class="QLabel" name="lowerLabel">
       <property name="text">
        <string>Lower:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="lowerHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Target:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>225</y>
     <width>205</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="alphaHLayout_bottom"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>204</y>
     <width>205</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="alphaHLayout_top">
    <item>
     <widget class="QLabel" name="alphaLabel">
      <property name="text">
       <string>Alpha</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="alphaHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>275</y>
     <width>205</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="replaceRunLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="replaceCheckBox">
      <property name="text">
       <string>Replace</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run Filter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>438</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="minMaxHLayout">
    <item>
     <widget class="QLabel" name="minLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>max</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="histogramView">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>308</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
